src/R/Editor/Impl/Roxygen/RoxygenBlock.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text;
+using Microsoft.Common.Core;
+using Microsoft.Languages.Core.Text;
+using Microsoft.Languages.Editor.Text;
+using Microsoft.R.Core.AST;
+using Microsoft.R.Core.AST.Arguments;
+using Microsoft.R.Core.AST.Extensions;
+using Microsoft.R.Core.AST.Functions;
+using Microsoft.R.Core.AST.Operators;
+using Microsoft.R.Core.AST.Variables;
+using static System.FormattableString;
+
+namespace Microsoft.R.Editor.Roxygen {
+    public static class RoxygenBlock {
```

Comment:
Yes, added support for S4. Looks like when file moves git loses diff tracking...